source.txt

```diff
@@ -157,7 +157,9 @@ cannot retrieve the document, nor discover its existence.
 
 This draft does not propose one universal solution, but rather lists
 what puzzle pieces we are aware of, and what options may exist to put
-them together. In general, we are interested in the case where there may
+them together. The authors expect this document could be used as a 
+guideline for an interoperable sharing between decentralized platforms.
```

Comment:
Hm but at this point we didn't talk about these monocultures yet. How about if we move it [here](https://github.com/Decentralized-Sharing-Working-Group/internet-draft/blob/master/source.txt#L146) and then write it as: 'The authors hope this document will contribute to more interoperability between such software applications.'?